cmd/dmsg-discovery/internal/api/api.go

```diff
@@ -294,6 +295,30 @@ func (a *API) getAvailableServers() http.HandlerFunc {
 	}
 }
 
+// getAllServers returns all servers entries as am array of json codified entry objects
```

Comment:
Change `entries as am array` to`entries as an array`

Reply:
Done.